### PR TITLE
added helper function for converting amounts to lot sized amount

### PR DESCRIPTION
**Pull request by barthr**

### CODE REVIEW

**Comment by adshao** [comment]
thanks @barthr

File: helpers_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+			want: 11.312,
+		},
+		{
+			name: "test with big number",
+			args: args{
+				lot:       0.0010000,
+				precision: 8,
+				amount:    11232821093480213.31232419283240912834434,
+			},
+			want: 11232821093480213.3123,
```
Comment (yeouchien) [inline helpers_test.go]: Shouldn't it be `11232821093480213.312`?

Comment (barthr) [reply helpers_test.go]: I think you are right, I have to look more into this

Comment (yeouchien) [reply helpers_test.go]: This is the formula I'm currently using:
`math.Floor(amount/lot) * lot`

Comment (barthr) [reply helpers_test.go]: That doesn't take into account the precision

**Pull request merged**